lib/reports/ReportCollisionDirectory.js

```diff
@@ -7,16 +7,19 @@ class ReportCollisionDirectory extends ReportBaseCrash {
     return ReportType.COLLISION_DIRECTORY;
   }
 
-  transformData(/* location, collisions, filters */) {
-    // TODO: implement this
+  transformData(location, rawData) {
+    return rawData;
```

Comment:
why does this `transformData()` method get to share references with the result of `ReportBaseCrash#fetchRawData`?  we should either change the documentation on `ReportBase#fetchRawData`, or we should make this deep-clone `rawData`.

Reply:
(actually, this will probably change as we move towards building the reports, so let's leave it for now)